Apps/Database/src/Delegates/DBNoteDelegate.cs

```diff
@@ -127,6 +127,10 @@ public DBResult<Note> UpdateNote(Note note, bool commit = true)
                 {
                     this.dbContext.SaveChanges();
                     result.Status = DBStatusCode.Updated;
+
+                    // Forces the refresh from the database.
+                    this.dbContext.Entry(note).State = EntityState.Detached;
+                    result.Payload = this.dbContext.Note.Find(note.Id, note.HdId);
```

Comment:
I have some concerns about the performance of this fix/approach. Normally in the UpdateRecord, we don't refresh anything from DB. I'm suggesting to look for a better approach.